<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="3" skipped="0" tests="28" time="15.474" timestamp="2025-10-31T05:48:04.461085-07:00" hostname="Snigdhas-MacBook-Pro.local"><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_get_transaction_by_userid" time="0.108" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_create_transaction" time="0.101" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_create_transaction_authentication_invalidtoken" time="0.101" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_get_transactions_not_exists" time="0.101" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_auth_transactions_insuffient_permission" time="0.101" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_user_detail_data_validation" time="0.102" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_create_transaction_empty_userid" time="0.101" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_create_transactionid_empty_receipient_id" time="0.101" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_create_transaction_id_invalid_amount" time="0.102" /><testcase classname="tests.api.test_transaction_API.TestUserApi" name="test_get_transaction_cross_user_access_check_auth_neg" time="0.102" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_get_users_by_id" time="0.102" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_create_users" time="0.102" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_create_users_authentication_invalidtoken" time="0.102" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_get_users_not_exists" time="0.101" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_auth_users_insuffient_permission" time="0.102" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_user_detail_data_validation" time="0.101" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_create_user_empty_name" time="0.101" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_create_user_invalid_email" time="0.101" /><testcase classname="tests.api.test_user_API.TestUserApi" name="test_get_users_server_error" time="0.102" /><testcase classname="tests.ui.transcation_creation_test.TestTransactionCreation" name="test_transaction_creation[trans0]" time="1.291" /><testcase classname="tests.ui.transcation_creation_test.TestTransactionCreation" name="test_transaction_creation_error[trans0]" time="5.393"><failure message="AssertionError: Locator expected to have text 'All fields are required and amount must be positive!'&#10;Actual value:  &#10;Call log:&#10;LocatorAssertions.to_have_text with timeout 5000ms&#10;  - waiting for locator(&quot;#message&quot;)&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;&#10;  -   locator resolved to &lt;div id=&quot;message&quot; class=&quot;hidden&quot;&gt;&lt;/div&gt;&#10;  -   unexpected value &quot;&quot;">tests/ui/transcation_creation_test.py:39: in test_transaction_creation_error
    expect(self.transaction_creation.get_message_locator()).to_have_text(trans['expected']['ui'])
E   AssertionError: Locator expected to have text 'All fields are required and amount must be positive!'
E   Actual value:  
E   Call log:
E   LocatorAssertions.to_have_text with timeout 5000ms
E     - waiting for locator("#message")
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""
E     -   locator resolved to &lt;div id="message" class="hidden"&gt;&lt;/div&gt;
E     -   unexpected value ""</failure></testcase><testcase classname="tests.ui.transcation_creation_test.TestTransactionCreation" name="test_transaction_creation_error[trans1]" time="1.194" /><testcase classname="tests.ui.transcation_creation_test.TestTransactionCreation" name="test_transaction_creation_error[trans2]" time="1.199" /><testcase classname="tests.ui.user_creation_test.TestUserCreation" name="test_user_creation[users0]" time="1.205" /><testcase classname="tests.ui.user_creation_test.TestUserCreation" name="test_user_creation[users1]" time="1.196" /><testcase classname="tests.ui.user_creation_test.TestUserCreation" name="test_user_creation_error[users0]" time="0.398"><failure message="ValueError: Unsupported type: &lt;class 'str'&gt;">tests/ui/user_creation_test.py:37: in test_user_creation_error
    expect(self.user_creation.get_toast_message(users['expected']['message'])).to_have_text(users['expected']['ui'])
.venv/lib/python3.9/site-packages/playwright/sync_api/__init__.py:142: in __call__
    raise ValueError(f"Unsupported type: {type(actual)}")
E   ValueError: Unsupported type: &lt;class 'str'&gt;</failure></testcase><testcase classname="tests.ui.user_creation_test.TestUserCreation" name="test_user_creation_error[users1]" time="1.198" /><testcase classname="tests.ui.user_creation_test.TestUserCreation" name="test_user_creation_error[users2]" time="0.383"><failure message="ValueError: Unsupported type: &lt;class 'str'&gt;">tests/ui/user_creation_test.py:37: in test_user_creation_error
    expect(self.user_creation.get_toast_message(users['expected']['message'])).to_have_text(users['expected']['ui'])
.venv/lib/python3.9/site-packages/playwright/sync_api/__init__.py:142: in __call__
    raise ValueError(f"Unsupported type: {type(actual)}")
E   ValueError: Unsupported type: &lt;class 'str'&gt;</failure></testcase></testsuite></testsuites>